cli: update Anbox installation output message

keirthana opened the pull request:
## Why is this needed?
The Anbox Cloud appliance installation message is missing a step before initialising it. Adding this to the CLI output so that users can follow the correct sequence of commands.

<!--
By default, we rebase PRs and will ask for a clean well-organized commit history in the PR before rebasing.
If your PR is small enough and you prefer, uncomment the following section and fill it out to request a squashed PR.
-->
<!--
## Please Squash this PR with this commit message

```
summary: no more than 70 characters

A description of what the change being made is and why it is being
made, if the summary line is insufficient.  The blank line above is
required. This should be wrapped at 72 characters, but otherwise has
no particular length requirements.

If you need to write multiple paragraphs, feel free.

LP: #NNNNNNN (replace with the appropriate Launchpad bug reference if applicable)
Fixes: #NNNNNNN (replace with the appropriate github issue if applicable)
```
-->

## Test Steps
1. Enable `anbox-cloud` on Ubuntu Pro
2. Verify if the output contains the updated text.

<!-- Example:
```
env SHELL_BEFORE=1 ./tools/test-in-lxd.sh xenial
# Set up test scenario before upgrade
exit # new version gets installed after exit and lxc shell is re-started
sudo pro new-sub-command --new-flag
# Assert something
```
-->


---

- [ ] *(un)check this to re-run the checklist action*

renanrodrigo commented on uaclient/messages/__init__.py:
```diff
@@ -1294,7 +1294,19 @@ class TxtColor:
+To finish setting up the Anbox Cloud Appliance, run the following commands
+sequentially:
+
+The following command lets you preview a script that installs some additional packages,
```
Not sure about the best way to reword it, but maybe avoid repeating "following command"?

keirthana replied:
@renanrodrigo Thank you, I have reworded it and also updated the `anbox.feature` file. Please have a look.

renanrodrigo reviewed (commented):
Hello, @keirthana , thanks for fixing this message.

The `features/anbox.feature` file contains a test which checks this message, and that needs to be updated too.

morphis commented on features/anbox.feature:
```diff
@@ -90,7 +90,19 @@ Feature: Enable anbox on Ubuntu
+
+      Once you have previewed the script, apply it to complete the installation:
+
+      $ anbox-cloud-appliance prepare-node-script | sudo bash -ex
```
Maybe we should do the following for a more secure way to not have the app changing what has been reviewed, e.g.

```
anbox-cloud-appliance prepare-node-script > prepare.sh
# Now review the script and once done, apply via
sudo bash -ex prepare.sh
```
@keirthana wdyt?

keirthana replied:
Thanks @morphis, that is indeed better. I have applied the feedback.

renanrodrigo reviewed (approved):
lgtm!

To be merged when we sort out the last release bits

keirthana commented:
> @keirthana hunspell is unhappy about `initialise` I remember seeing something about switching to en_US - and tbh the client messages are en_US anyway
> 
> I would say this needs to read `initialize` instead?

Thanks @renanrodrigo I thought the Ubuntu Pro client docs were still on en_GB. I have fixed it now. 

I also fixed the URL for the initialize section in the `urls.py` which makes the formatting check complain. I don't know if it complains because there is a limit on the number of characters or something else is amiss. Could you help?

keirthana commented:
> > @keirthana hunspell is unhappy about `initialise` I remember seeing something about switching to en_US - and tbh the client messages are en_US anyway
> > I would say this needs to read `initialize` instead?
> 
> Thanks @renanrodrigo I thought the Ubuntu Pro client docs were still on en_GB. I have fixed it now.
> 
> I also fixed the URL for the initialize section in the `urls.py` which makes the formatting check complain. I don't know if it complains because there is a limit on the number of characters or something else is amiss. Could you help?

I tried reformatting the file but now style check complains that the line is too long. :woman_shrugging:

renanrodrigo commented:
@keirthana yep it happens - that's a big URL heh

please keep whatever black defined, and add `  # noqa: E501` (mind the couple spaces) after the end of the string, so it's ignored by the style check

keirthana commented:
> @keirthana yep it happens - that's a big URL heh
> 
> please keep whatever `black` defined, and add ` # noqa: E501` (mind the couple spaces) after the end of the string, so it's ignored by the style check

Thanks, that helped!

renanrodrigo commented:
Thanks!
